### PR TITLE
Add Navi32 TN_HHS/NN_HHS Kernels

### PR DESCRIPTION
**Pull request by wenchuanchen**
same as https://github.com/ROCm/hipBLASLt/pull/1133
for these two tuning requests:
https://ontrack-internal.amd.com/browse/SWDEV-480801
https://ontrack-internal.amd.com/browse/SWDEV-480799

### CODE REVIEW

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs.yaml [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+    SolutionNameMin: Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs_MT64x96x32_MI16x16x1_SN_LDSB0_GRVWA8_GRVWB8_GSU2_GSUC0_GSUWGMRR0_K1_LBSPPA2048_LBSPPB128_MIWT2_3_NLCA1_SS1_SU32_SUM0_SUS256_TLDS1_WG32_4_1_WGM8
+    SourceSwap: 1
+    StaggerU: 32
+    StaggerUMapping: 0
+    StaggerUStride: 256
+    StorePriorityOpt: false
+    StoreRemapVectorWidth: 0
+    StoreSyncOpt: 0
+    StoreVectorWidth: 1
+    StreamK: 0
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs.yaml:584]: some parameter are not in 6.3 release staging branch yet.

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs.yaml [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+  UseInitialStridesAB: false
+  UseInitialStridesCD: false
+  UseScaleAB: ''
+  UseScaleAlphaVec: 1
+  UseScaleCD: false
+- - 1LDSBuffer: 0
+    ActivationAlt: false
+    ActivationFuncCall: true
+    ActivationFused: true
+    AssertAIGreaterThanEqual: -1
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs.yaml]: 6.3 branch doesn't have this paramter

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs.yaml (hunk 1 above)
Comment (wenchuanchen) [reply library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs.yaml:584]: No impact on installing

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs.yaml (hunk 2 above)
Comment (wenchuanchen) [reply library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_HHS_BH_Bias_AS_SAV_UserArgs.yaml]: No impact on installing

**Review by cmingch — APPROVED** [review]

**Review by jichangjichang — APPROVED** [review]
lgtm

**Comment by wenchuanchen** [comment]
[tox.log](https://github.com/user-attachments/files/17124197/tox.log)
[test.log](https://github.com/user-attachments/files/17124201/test.log)
hipblaslt-test i8 failed cases are know issue and not relevant to this PR

**Pull request merged**